decidim-core/app/validators/uploader_image_dimensions_validator.rb

```diff
@@ -29,6 +29,18 @@ def validate_image_size(record, attribute, file, uploader)
     return if (image = extract_image(file)).blank?
 
     record.errors.add attribute, I18n.t("carrierwave.errors.file_resolution_too_large") if image.dimensions.any? { |dimension| dimension > uploader.max_image_height_or_width }
+  rescue MiniMagick::Error
```

Comment:
In order to fix some of the specs I think we need to modify this slightly:
```suggestion
  rescue MiniMagick::Error, MiniMagick::Invalid
```

For the others specs, we might need to modify the specs themselves to make it work under Ubuntu 20.04.

Reply:
@alecslupu can you check this out 🙏🏽 ? Thanks